config/routes.rb

```diff
@@ -23,6 +23,15 @@
         post 'resend'
       end
     end
+    resources :phases, only: [] do
+      resources :evaluator_submissions, only: [:index] do
+        member do
+          post 'unassign'
+          post 'reassign'
+        end
+      end
+      get 'evaluator_submissions/:evaluator_id', to: 'evaluator_submissions#index', as: :evaluator_submissions_for_evaluator
```

Comment:
this might work better renamed to `evaluator_submission_assignments` resource with normal REST/crud routes. I don't think we will need the `:show` route for individual assignments so that one can use evaluator_id in the id param to show the assignments for a particular evaluator, e.g. `/phases/:phase_id/evaluator_submission_assignments/:id`
```suggestion
      resources :evaluator_submission_assignments, only: [:index, :show, :create, :destroy]
```
can both the assign and reassign actions use the create route?

Reply:
ig we could go with `evaluator_assignments` or `submission_assignments` for more brevity here but I feel like `evaluator_submissions` is confusing